### PR TITLE
feat(toggles): add plugin policy toggles

### PR DESCRIPTION
**Pull request by slonka**
alternative to https://github.com/kumahq/kuma/pull/8760

### Checklist prior to review

- [x] [Link to relevant issue][1] as well as docs and UI issues --
- [ ] This will not break child repos: it doesn't hardcode values (.e.g "kumahq" as a image registry) and it will work on Windows, system specific functions like `syscall.Mkfifo` have equivalent implementation on the other OS --
- [ ] Tests (Unit test, E2E tests, manual test on universal and k8s) --
  - Don't forget `ci/` labels to run additional/fewer tests
- [ ] Do you need to update [`UPGRADE.md`](../blob/master/UPGRADE.md)? --
- [ ] Does it need to be backported according to the [backporting policy](../blob/master/CONTRIBUTING.md#backporting)? ([this](https://github.com/kumahq/kuma/actions/workflows/auto-backport.yaml) GH action will add "backport" label based on these [file globs](https://github.com/kumahq/kuma/blob/master/.github/workflows/auto-backport.yaml#L6), if you want to prevent it from adding the "backport" label use [no-backport-autolabel](https://github.com/kumahq/kuma/blob/master/.github/workflows/auto-backport.yaml#L8) label) --

[1]: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

### CODE REVIEW

File: pkg/plugins/policies/imports.go [hunk 1]
```diff
@@ -1,18 +1,61 @@
...
+		}
+	}
+}
+
+func InitPolicies(enabledPluginPolicies []string) {
+	for _, policy := range enabledPluginPolicies {
+		initializer, ok := nameToModule[policy]
+		if ok && !initializer.initialized {
+			initializer.initFn()
+			initializer.initialized = true
```
Comment (slonka) [inline pkg/plugins/policies/imports.go:56]: This is not nice but needed, we initialise the policies in both `RunSpec` and `kumactl` code and if both are reached by test code then we get panics on double registration. I don't see a nice way around this, suggestions welcomed.

File: docs/generated/raw/kuma-cp.yaml [hunk 2]
```diff
@@ -753,19 +730,32 @@ experimental:
...
   openTelemetry:
     endpoint: "" # e.g. otel-collector:4317
-
 # Configuration of the event bus which is local to one instance of CP
 eventBus:
   # BufferSize controls the buffer for every single event listener.
   # If we go over buffer, additional delay may happen to various operation like insight recomputation or KDS.
   bufferSize: 100 # ENV: KUMA_EVENT_BUS_BUFFER_SIZE
+policies:
+  pluginPoliciesEnabled:
```
Comment (jakubdyszkiewicz) [inline docs/generated/raw/kuma-cp.yaml:747]: Comment + ENV: ?

**Review by jakubdyszkiewicz — APPROVED** [review]

File: pkg/config/plugins/policies/policies.go [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package policies
+
+func DefaultPoliciesConfig() *Config {
+	return &Config{
+		PluginPoliciesEnabled: []string{
+			"meshaccesslogs",
```
Comment (jijiechen) [inline pkg/config/plugins/policies/policies.go:6]: Is it better to make these strings constants?

**Review by jijiechen — APPROVED** [review]

File: pkg/config/plugins/policies/policies.go (hunk 3 above)
Comment (slonka) [reply pkg/config/plugins/policies/policies.go:6]: not sure it matters that much, I already have this in this format and it's autogenerated so would require more changes. If you really want to change it please open up an issue.

**Review by jakubdyszkiewicz — APPROVED** [review]

**Pull request merged**